test/functional/tests/security/test_security_user.py

```diff
@@ -0,0 +1,458 @@
+        casadm_file = fs_utils.parse_ls_output(fs_utils.ls_item(f"{casadm_path}"))[0]
+        casadm_copy = casadm_file.copy(casadm_dest_path, True)
+        casadm_copy.chmod_numerical(777)
+
+    with TestRun.step("Unmount core."):
+        core.unmount()
+
+    with TestRun.step("Add non-root user account."):
+        TestRun.executor.run(f"useradd -N -r -l {user_name}")
+        user_home_dir = fs_utils.parse_ls_output(fs_utils.ls_item(f"/home/{user_name}"))[0]
+        user_home_dir.chmod_numerical(777, True)
+
+    with TestRun.step("Try to stop OpenCAS service."):
+        try:
+            run_as_other_user(cli.ctl_stop(False), user_name)
+        except CmdException:
+            TestRun.LOGGER.info("Non-root user cannot stop OpenCAS.")
+
+    with TestRun.step("Try to start OpenCAS service."):
+        try:
+            run_as_other_user(cli.ctl_start(), user_name)
+        except CmdException:
+            TestRun.LOGGER.info("Non-root user cannot start OpenCAS.")
+
+    with TestRun.step("Stop OpenCAS."):
+        casadm.stop_all_caches()
+
+    with TestRun.step("Try to init OpenCAS service."):
+        try:
+            run_as_other_user(cli.ctl_init(True), user_name)
+        except CmdException:
+            TestRun.LOGGER.info("Non-root user cannot init OpenCAS.")
+
+    with TestRun.step("Add non-root user account to sudoers group."):
+        TestRun.executor.run(f'echo "{user_name} ALL = (root) NOPASSWD:ALL" '
+                             f'| sudo tee/etc/sudoers.d/{user_name}')
+
+    with TestRun.step("Try to stop OpenCAS service with 'sudo'."):
+        try:
+            run_as_other_user(cli.ctl_stop(False), user_name, True)
+        except CmdException:
+            TestRun.fail("Non-root sudoer user should can stop OpenCAS.")
+
+    with TestRun.step("Try to start OpenCAS service with 'sudo'."):
+        try:
+            run_as_other_user(cli.ctl_start(), user_name, True)
+        except CmdException:
+            TestRun.fail("Non-root sudoer user should can start OpenCAS.")
+
+    with TestRun.step("Stop OpenCAS."):
+        casadm.stop_all_caches()
+
+    with TestRun.step("Try to stop init OpenCAS service with 'sudo'."):
+        try:
+            run_as_other_user(cli.ctl_init(True), user_name, True)
+        except CmdException:
+            TestRun.fail("Non-root sudoer user should can init OpenCAS.")
+
+    with TestRun.step("Remove user account."):
+        TestRun.executor.run(f"userdel -r -Z {user_name}")
+
+    with TestRun.step("Stop all caches."):
+        casadm.stop_all_caches()
+
+
+def run_as_other_user(command, user: str, sudo: bool = False):
+    prefix = f'sudo -u {user}'
+    if sudo:
+        command = 'sudo ' + command
+    command = f'{prefix} {command}'
+    output = TestRun.executor.run(command)
+    if output.exit_code != 0 or output.stderr is not "":
+        raise CmdException("Must be run as root.", output)
+    return output
```

Comment:
Moved here my function to avoid placing useless for others things in test-framework.